ftests/parallel/test_remote.py

```diff
@@ -129,15 +129,21 @@ def test_dsoln(self):
         from nose.plugins.skip import SkipTest
         if sys.platform.startswith('win'): raise SkipTest
         from numpy import zeros
-        from solvcon.batch import Localhost
         from solvcon.domain import Distributed
+        if "SOLVCON_NODELIST" in os.environ:
```

Comment:
> A test case should behave consistently even when the environment changes.

You convinced me of this point of view. Let me close this PR and create another one to have a standalone test case.